src/GeositeFramework/css/main.css

```diff
@@ -2335,3 +2335,7 @@ body.x-body {
 .launchpad-plugin li p {
   font-size: 13px;
 }
+.btn-default.dropdown-toggle#map-utils-dropdown-button > .icon-wrench:before {
```

Comment:
Sounds good to me! I just rebased this on `prototype` then pushed a fixup which uses that selector to get the icon.